cypress/cypress.config.js

```diff
@@ -0,0 +1,123 @@
+const {execSync, spawn} = require("child_process");
+const path = require("path");
+
+const isLocalRun = process.env["CYPRESS_IS_LOCAL_RUN"] || false;
+
+function getWiremockPath(filename) {
```

Comment:
⚠️ cypress expects the `cypress.config.js` file to live in the root of the project. since it exists in our `cypress` directory I had to add this to make sure the filepath is valid for local and docker runs